### ISSUE TITLE
yunohost app list is broken in 11.0.0~alpha+202106220853

### ISSUE THREAD

**Issue by yalh76** [YunoHost/issues#1831]
```
yunohost tools versions
yunohost:
  repo: unstable
  version: 11.0.0~alpha+202106220853
yunohost-admin:
  repo: unstable
  version: 11.0.0~alpha+202103081705
moulinette:
  repo: unstable
  version: 11.0.0~alpha+202105272317
ssowat:
  repo: unstable
  version: 11.0.0~alpha+202102050117
```

```
yunohost app list
Traceback (most recent call last):
  File "/usr/bin/yunohost", line 66, in <module>
    yunohost.cli(
  File "/usr/lib/moulinette/yunohost/__init__.py", line 25, in cli
    ret = moulinette.cli(args, output_as=output_as, timeout=timeout, top_parser=parser)
  File "/usr/lib/python3/dist-packages/moulinette/__init__.py", line 119, in cli
    Cli(top_parser=top_parser, load_only_category=load_only_category).run(
  File "/usr/lib/python3/dist-packages/moulinette/interfaces/cli.py", line 502, in run
    ret = self.actionsmap.process(args, timeout=timeout)
  File "/usr/lib/python3/dist-packages/moulinette/actionsmap.py", line 600, in process
    return func(**arguments)
TypeError: app_list() got an unexpected keyword argument 'installed'
```

Don't know if it's the good place to declare it, feel free to close it.

**Comment by alexAubin** [comment]
Ah yes indeed, thanks for the report, should be fixed by https://github.com/YunoHost/yunohost/commit/a274b6a8fca4ceebac9f685b57d862c4554be67f

**Comment by yalh76** [comment]
Thanks a lot, It's fixed in last alpha: 11.0.0~alpha+202107031617 ^^
